corehq/apps/app_manager/templates/app_manager/v2/form_designer.html

```diff
@@ -81,6 +82,11 @@
                     {% if request.guided_tour %}
                     form_tour_start();
                     {% endif %}
+                    if ($('.fd-question-tree li').length > 0) {
```

Comment:
Did it not work out to do this in a less fragile way, like checking server-side if the form has questions?